CONTRIBUTING.md

```diff
@@ -272,13 +290,16 @@ repository] first.
 number it relates to as well as a quick explanation of what the branch is
 about:
 
-        $ git checkout -b 123456-invalid-session-content
+    ```bash
+    git checkout -b 123456-invalid-session-content
+    ```
 
 1. Work on your changes, test them, iterate, commit your work.
 
 1. Before sending your changes for review, make sure to rebase your work using
    the most up-to-date data from the main repository:
 
+    ```bash
```

Comment:
Uniformity, remove space?

Reply:
but this codeblock has to be a part of the numbered list item.